src/utils/config_utils/src/config_utils.c

```diff
@@ -20,6 +20,7 @@
 #include <stdbool.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
```

Comment:
Remove if possible; otherwise, needs to be aducpal/unistd.h

Reply:
Removed